### PR TITLE
Add autofix to value-list-comma-newline-after

### LINKED ISSUE #3595: Add autofix to value-list-comma-newline-after
> What is the problem you're trying to solve?

Adds autofix of rules [defined in `stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js) so that everyone can start using [`stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard) easily.

> What solution would you like to see?

Add autofix to [value-list-comma-newline-after](https://github.com/stylelint/stylelint/tree/master/lib/rules/value-list-comma-newline-after)

### PR DESCRIPTION
**Pull request by littlemyx**
> Which issue, if any, is this issue related to?

Closes #3595

> Is there anything in the PR that needs further explanation?

ex.

- option: always***

 code:

 ```css
 a { background-size: 0,0; }
 ```
 fixed:

 ```css
 a { background-size: 0,
      0; }
 ```
- option: never-multi-line

 code:
 ```css
 a { background-size: 0
       , 0; }
 ```

 fixed:
 ```css
 a { background-size: 0
      ,0; }
 ```

### CODE REVIEW

File: lib/rules/value-list-comma-newline-after/index.js [hunk 1]
```diff
@@ -26,12 +27,49 @@ const rule = function(expectation) {
...
+            commaIndices.push(index);
+            fixData.set(declNode, commaIndices);
+            return true;
+          }
+        : null
     });
+    if (fixData) {
+      fixData.forEach((commaIndices, decl) => {
+        commaIndices
+          .sort()
```
Comment (ota-meshi) [inline lib/rules/value-list-comma-newline-after/index.js]: There was a bug in `value-list-comma-space-after`, can you make the same change?
https://github.com/stylelint/stylelint/pull/3621/files#diff-812c870b12495af18b84bb163acd432bL54

**Comment by littlemyx** [comment]
So, is there any other problems with PR?

**Comment by ntwb** [comment]
I've restarted he AppVeyor build...

**Comment by littlemyx** [comment]
Sooo...?)

**Comment by jeddy3** [comment]
>    Added: `value-list-comma-newline-after` autofix ([#3616](https://github.com/stylelint/stylelint/pull/3616)).

**Pull request merged**